sdk/translation/Azure.AI.Translation.Document/CHANGELOG.md

```diff
@@ -5,11 +5,12 @@
 ### Features Added
 - Added optional parameter `categoryId` to the `DocumentTranslationInput.AddTarget`.
 - Added property `Ascending` to type `DocumentFilterOrder` and `TranslationFilterOrder`.
-- Renamed parameter `asc` to `ascending` in `TranslationFilterOrder` constructor.
+- `DocumentTranslationAudience` has been added to allow the user to select the Azure cloud where the resource is located.
```

Comment:
Do we have any pipelines enabled for national cloud testing is this something we should add to the backlog?

Reply:
Just saw Krista had asked the same question.

Reply:
https://github.com/Azure/azure-sdk-for-net/issues/29000